examples/single-sign-on/iam-identity-center/README.md

```diff
@@ -0,0 +1,129 @@
+# IAM Identity Center Single Sign-On for Amazon EKS Cluster
+
+These example demonstrates how to deploy an Amazon EKS cluster that is deployed on the AWS Cloud, integrated with IAM Identity Center (former AWS SSO) as an the Identity Provider (IdP) for Single Sign-On (SSO) authentication. The authorization configuration layer still being done using Kubernetes Role-based access control (RBAC). By the this time we have integration with the following IdPs.
+
+## Prerequisites:
+
+Ensure that you have the following tools installed locally:
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+Also make sure you have enabled the following AWS resource:
+
+1. Enable [IAM Identity Center](https://console.aws.amazon.com/singlesignon/home/)
+    This will also create an AWS Organization in your account.
+
+## Deploy
+
+To provision these examples, run the following commands:
+
+```sh
+terraform init
+terraform apply -target module.vpc
+terraform apply -target module.eks
+terraform apply
+```
+
+Enter `yes` at command prompt to apply
+
+## Validate
+
+After the `terraform` commands are executed sucessfully, check if the newly created users are active.
+
+To do that use the link provided in the email invite if you added a valid email address for your users, or go to the [IAM Identity Center Console](https://console.aws.amazon.com/singlesignon/home/), in the *Users* dashboard on the left hand side menu, then select the user, and click on *Reset password* button on the upper right corner. Choose the option to *Generate a one-time password and share the password with the user*.
```

Comment:
Where should I add email addresses for my users? I am not sure its clear if this is something I do via Terraform or AWS Console.

Reply:
Fixed.